src/components/Feedback/FeedbackModal.tsx

```diff
@@ -42,19 +41,18 @@ export type FeedbackPages =
 
 const FeedbackModal = memo(() => {
   const intl = useIntl();
-  const usePendoFeedback = useSelector<ReduxState, boolean | undefined>(({ chrome: { usePendoFeedback } }) => usePendoFeedback);
-  const isOpen = useSelector<ReduxState, boolean | undefined>(({ chrome: { isFeedbackModalOpen } }) => isFeedbackModalOpen);
-  const dispatch = useDispatch();
+  const [isModalOpen, setIsModalOpen] = useAtom(isFeedbackModalOpenAtom);
+  const [usePendoFeedback] = useAtom(usePendoFeedbackAtom);
```

Comment:
You can use `useAtomValue` if you are not interested in the state update function.